hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/scheduler/SchedulerService.java

```diff
@@ -58,14 +66,17 @@ public SchedulerService(
         // metrics for our own internals
         this.diagnostics = diagnostics;
 
-        // create the schedulers - we use two: one for metric collections and one for avail checks
+        // create the schedulers - we use three: one for metric collections, one for avail checks and one for feed pings
         this.metricStorage = new MetricBufferedStorageDispatcher(configuration, storageAdapter, diagnostics);
         this.metricScheduler = MeasurementScheduler.forMetrics("Hawkular-WildFly-Agent-Scheduler-Metrics",
                 metricStorage);
 
         this.availStorage = new AvailBufferedStorageDispatcher(configuration, storageAdapter, diagnostics);
         this.availScheduler = MeasurementScheduler.forAvails("Hawkular-WildFly-Agent-Scheduler-Avail",
                 availStorage);
+
+        this.pingStorage = new PingStorageDispatcher(configuration, storageAdapter, diagnostics);
+        this.pingScheduler = new ScheduledThreadPoolExecutor(1);
```

Comment:
pass in a thread factory here so we can give the thread a name (the default will be a generic "ThreadPool-01" or something like that, and its hard to see it in thread dumps.) and make it a daemon thread so it won't hang up the VM exiting if it gets stuck. So something like:

        ThreadFactory threadFactory = ThreadFactoryGenerator.generateFactory(true, "Hawkular-WildFly-Scheduler-Ping");

And pass that thread factory to the ScheduledThreadPoolExecutor constructor